### PR TITLE
Simplify SetForegroundWindow

### PR DESCRIPTION
**Pull request by sharwell**
Switch from `SetForegroundWindow` to `SwitchToThisWindow` to bring a window to the front. The latter is used by Task Manager's "Switch To" functionality, and doesn't have the same restrictions regarding the state of the window manager prior to the call.

### CODE REVIEW

File: src/VisualStudio/IntegrationTest/IntegrationTests/VisualBasic/BasicEditAndContinue.cs [hunk 1]
```diff
@@ -209,7 +209,7 @@ End Module
 ");
             VisualStudio.Workspace.WaitForAsyncOperations(FeatureAttribute.Workspace);
             VisualStudio.Debugger.Go(waitForBreakMode: false);
-            VisualStudio.ActivateMainWindow(skipAttachingThreads: true);
```
Comment (jaredpar) [inline src/VisualStudio/IntegrationTest/IntegrationTests/VisualBasic/BasicEditAndContinue.cs]: Why was attaching threads a problem?

Comment (jasonmalinowski) [reply src/VisualStudio/IntegrationTest/IntegrationTests/VisualBasic/BasicEditAndContinue.cs]: Agreed, can we update the commit description to explain if we think this is ready?

Comment (sharwell) [reply src/VisualStudio/IntegrationTest/IntegrationTests/VisualBasic/BasicEditAndContinue.cs]: Unfortunately it is not something I have the ability to explain.

**Comment by sharwell** [comment]
Did not work

**Comment by sharwell** [comment]
@jinujoseph for approval

**Comment by sharwell** [comment]
Merging this **test only** change to help with infrastructure reliability

**Pull request merged**